static/js/directives/ui/manifest-vulnerability-view.js

```diff
@@ -29,7 +29,10 @@ angular.module('quay').directive('manifestVulnerabilityView', function () {
       };
 
       $scope.getDistro = function(vuln) {
-        return vuln['namespace'].split(':', 1);
+        if (vuln['namespace']) {
```

Comment:
Is this directly related to surfacing the enrichment data?

Reply:
I saw this failing on the UI when the key "namespace" is empty in the vuln object. This is part of the UI to display distribution names in manifest vulnerabilities.